test_framework/scripts/longhorn-setup.sh

```diff
@@ -393,11 +396,16 @@ main(){
       customize_longhorn_manifest_for_airgap
       install_longhorn_by_manifest "${TF_VAR_tf_workspace}/longhorn.yaml"
     elif [[ "${LONGHORN_INSTALL_METHOD}" == "helm-chart" ]]; then
-      create_registry_secret
+      if [[ "${CREATE_PRIVATE_REGISTRY_SECRET}" == false ]]; then
+        create_registry_secret
+      fi
       get_longhorn_chart
       customize_longhorn_chart_for_airgap
       install_longhorn_by_chart
     elif [[ "${LONGHORN_INSTALL_METHOD}" == "rancher" ]]; then
+      if [[ "${CREATE_PRIVATE_REGISTRY_SECRET}" == false ]]; then
```

Comment:
Ok, this just leaves to the chart installation which will use the default value `true` to create the secret instead. So when running for 1.2, we should select `false`. I feel this is quite complicated to understand. Can we just make it simple to implicitly make it take effect.

Can we just leave the secret only created during the chart installation instead of creating before that? so we will not need to differentiate different cases here? WDYT?